packages/devtools-reps/src/reps/grip-array.js

```diff
@@ -229,4 +266,5 @@ module.exports = {
   supportsObject,
   maxLengthMap,
   getLength,
+  getMoreEllipsisElement,
```

Comment:
why do we need to export this function ? It's already imported from utils.

Reply:
there's no need actually, it just used to be defined in `grip-array`. I'll remove this.